Games/types/Mafia/roles/Village/Fabulist.js

```diff
@@ -1,9 +1,7 @@
-const Role = require("../../Role");
+const Mailman = require("../../Mailman");
 
-module.exports = class Fabulist extends Role {
+module.exports = class Fabulist extends Mailman {
```

Comment:
I'm not a fan of this method of renaming roles. 

On the frontend, people hovering over the Courier icon will still see the old Courier description, though the role functionality is Mailman.

Deleting roles and adding them to renamed roles forces the role description to be blank, which is the nudging needed to clean up setups.

it's up to you. What vastined did was to rename roles on the frontend as well, but it contributes to role bloat overtime. E.g. with that method now and our current changes, we would have 30 extra roles sitting around in the code.

Reply:
I get what you mean. We should be as efficient as possible, avoid bloat. There's already certain cards that are redundant after so many changes, and others can be cleaned up and merged using different methods. Less code = runs faster.

I'll clean up my PR. Found alternate uses for Courier and Messenger while talking to users, so those roles will only be partially deprecated. Will work on this later.